Adding QuickNode faucet to docs

Sahilsen opened the pull request.

Ishmaello reviewed (commented):
https://ishmaello.solana-mainnet.discover.quiknode.pro/0c3f349f30884a9995f5e56b30eec975433e08fd/

ashucoder9 commented on docs/develop/tools/quicknode-faucet.md:
```diff
@@ -0,0 +1,34 @@
+
+:::tip
+
+If you Tweet with the [drip](https://faucet.quicknode.com/drip) faucet URL along with your address and submit the tweet URL, you get 10X more MATIC.
```
```suggestion
If you **tweet** the [<ins>drip faucet URL</ins>](https://faucet.quicknode.com/drip) along with your wallet address and submit the tweet URL, you get 10X more MATIC testnet tokens.
```

ashucoder9 commented on docs/develop/tools/quicknode-faucet.md:
```diff
@@ -0,0 +1,34 @@
+
+:::
+
+To receive MATIC Mumbai tokens, head to [QuickNode Multi-chain Faucet](https://faucet.quicknode.com/drip). Connect your wallet (MetaMask or Phantom) by clicking on **Connect Wallet**. Then select Polygon and Mumbai testnet from the dropdown menus, then click on **Continue**.
```
```suggestion
To receive MATIC Mumbai tokens, head to [QuickNode Multi-chain Faucet](https://faucet.quicknode.com/drip). Connect your wallet (MetaMask or Phantom) by clicking on **Connect Wallet**. Select Polygon and Mumbai Testnet from the dropdown menus, and then click on **Continue**.
```

ashucoder9 commented on docs/develop/tools/quicknode-faucet.md:
```diff
@@ -0,0 +1,34 @@
+![img](/img/tools/quicknode-faucet1.png)
+![img](/img/tools/quicknode-faucet2.png)
+
+To receive 0.1 testnet MATIC click on **No thanks, just send me 0.1 MATIC** or to receive 1 MATIC do a tweet with the [drip](https://faucet.quicknode.com/drip) faucet URL along with your address, copy-paste your tweet URL in the input field and click on **Claim my 1 MATIC**
```
```suggestion
To receive 0.1 Testnet MATIC, click on **No thanks, just send me 0.1 MATIC**. To receive 1 MATIC, post a tweet with the [drip faucet URL](https://faucet.quicknode.com/drip) along with your wallet address, and copy-paste your tweet URL in the input field and click on **Claim my 1 MATIC**.
```

ashucoder9 reviewed (changes requested):
Hey @Sahilsen, thanks for the contribution. Added a few suggestions.

Also, can you please update the screenshots with more uniform image capturing such that the size and content padding are consistent? If not, we can go with the merge and our team will update them during regular docs audit.

Sahilsen commented:
> Hey @Sahilsen, thanks for the contribution. Added a few suggestions.
> 
> Also, can you please update the screenshots with more uniform image capturing such that the size and content padding are consistent? If not, we can go with the merge and our team will update them during regular docs audit.

Hey @ashucoder9 thanks for the review. Changed the images, and now they should be all same size and also changes the copy according to your feedback.

ashucoder9 reviewed (approved):
Thanks @Sahilsen 🙌🏽

Sahilsen commented:
> Thanks @Sahilsen 🙌🏽

Are we all good to merge? @ashucoder9

ashucoder9 commented:
Yeah, you may merge the PR.

Sahilsen commented:
> Yeah, you may merge the PR.

I don't think I can merge, can you please? @ashucoder9

Ishmaello commented:
Neither do I be able to merge
